setup.py

```diff
@@ -4,17 +4,13 @@
 import os
 
 from distutils.command.sdist import sdist
-import setuptools
```

Comment:
IIRC, the import monkeypatches distutils and changes behavior.
Is it ok to remove?